packages/core/frontend/sirius-components-core/src/workbench/WorkbenchMachine.ts

```diff
@@ -170,9 +148,9 @@ export const workbenchMachine = Machine<WorkbenchContext, WorkbenchStateSchema,
 
           const newSelectedRepresentations = [...representations, ...newRepresentations];
 
-          return { selection, displayedRepresentation, representations: newSelectedRepresentations };
+          return { displayedRepresentation, representations: newSelectedRepresentations };
```

Comment:
It seems we're losing the test we had before which prevented "changing" the state with the same selected representations.